case-studies/policy/cli.md

```diff
@@ -43,81 +45,148 @@ The create command is used to create resources by filename.
+  bbfw create gnp -f policy.yaml
 
   # Create many global network policy
-  bbfwcli create gnp policy1.yaml policy2.yaml
+  bbfw create gnp -f policy1.yaml -f policy2.yaml
 
 Flags:
   -f, --file stringArray   file to read
   -h, --help               help for create
 ```
 
+## List resource
+```shell
+List resource
+
+Usage:
+  bbfw list [flags]
+
+Examples:
+  # List global network policy
+  bbfw list gns
+
+  # List global network policy
+  bbfw list gnp
+
+  # List global network policy with order
+  bbfw list gnp --isOrder
+
+  # List host endpoint
+  bbfw list hep
+
+  # List host endpoint with tenantID
+  bbfw list hep --tenantID=1
+
+  # List host endpoint with IP
+  bbfw list hep --ip=192.168.0.1
+
+  # List host endpoint with tenantID and IP
+  bbfw list hep --tenantID=1 --ip=192.168.0.1,
+
+
+Flags:
+  -h, --help            help for list
+      --ip string       Host Endpoint: filter by IP
+      --isOrder         Global Network Policy: filter by Order
+      --tenantID uint   Host Endpoint: filter by TenantID
+```
+
 ## Get resource
 ```shell
-Get resource by name
+Get resource
 
 Usage:
-  bbfwcli get [flags]
+  bbfw get [flags]
 
 Examples:
   # Get a global network policy by name
-  bbfwcli get gnp allow_ssh
+  bbfw get gnp allow_ssh
 
   # Get a global network policy by name with json output format
-  bbfwcli get gnp allow_ssh -o json
+  bbfw get gnp allow_ssh -o json
+
+ # Get a host endpoint
+  bbfw get hep --tenantID=1 --ip=192.168.123.0
+
+  # Get a global network set by name
+  bbfw get gns allow_ssh
+
+  # Get a global network set by name with json output format
+  bbfw get gns my_set -o json
 
 
 Flags:
   -h, --help            help for get
+      --ip string       HEP: get by ip
   -o, --output string   output format(yaml|json). Default: yaml
+      --tenantID uint   HEP: get by tenantID
 ```
 
 ## Delete resource
 
 ```shell
-The delete command is used to delete resources by name or filename.
+The delete command is used to delete resources by name(Global Network Policy, Global Network Set),
+by tenantID,IP(Host Endpoint) or filename. 
 
   Resource type available:
     * HostEndpoint(or hep)
     * GlobalNetworkSet(or gns)
     * GlobalNetworkPolicy(or gnp)
 
 Usage:
-  bbfwcli delete [resourceType] [flags]
+  bbfw delete [resourceType] [flags]
 
 Examples:
   # Delete a policy with name
-  bbfwcli delete gnp allow_ssh
+  bbfw delete gnp allow_ssh
+
+  # Delete many policy with name
+  bbfw delete gnp allow_ssh allow_ping
+
+  # Delete many policy with filename
+  bbfw delete gnp -f allow_ssh.yaml -f allow_ping.yaml
+
+  # Delete a set with name
+  bbfw delete gns server
 
   # Delete many policy with name
-  bbfwcli delete hep allow_ssh allow_ping
+  bbfw delete gns server vm
 
   # Delete many policy with filename
-  bbfwcli delete hep allow_ssh.yaml allow_ping.yaml
+  bbfw delete gns -f server.yaml -f vm.yaml
+
+  # Delete a hep with tenantID and ip
+  bbfw delete hep --tenantID=1 --ip=192.168.1.1
```

Comment:
I often misread "hep" as "help". What do you think if we use "endpoints"?

Reply:
if use endpoints, we will confuse endpoints for what. hep is abbreviation of hostendpoint. You can use bbfw delete hostendpoint